docs/pages/setup/reference/backends.mdx

```diff
@@ -474,7 +474,11 @@ teleport:
 
 ### DynamoDB autoscaling
 
-When setting up DynamoDB it's important to set up backup and autoscaling. We make setup simpler by allowing AWS DynamoDB settings to be set automatically during Teleport startup.
+When setting up DynamoDB it's important to set up backup. Autoscaling is optional with Teleport,
+it is advised to collect some production usage data before enabling autoscaling.
+
+Autoscaling and backup can be configured on DynamoDB directly. We also make setup simpler by allowing AWS DynamoDB
```

Comment:
What does "directly" mean here? Via the AWS console/CLI?

Reply:
It meant user would have to create an autoscaling policy. They can do it using CLI, web console, cloudformation, or any existing CDK. I rephrased to focus on what to create instead of how to create it.